ios/native/NativeProxy.mm

```diff
@@ -93,7 +93,7 @@ + (void*) getNativeReanimatedModule:(void*)jsInvokerVoidPtr
 + (NSArray<NSArray*>*)getChangedSharedValues
 {
   NSMutableArray *changed = [NSMutableArray new];
-  for(auto & sharedValue : nativeReanimatedModule->sharedValueRegistry->sharedValueMap) {
+  for(auto & sharedValue : nativeReanimatedModule->sharedValueRegistry->getSharedValueMap()) {
```

Comment:
Shouldn't we make the same change in NativeProxy.cpp?

Reply:
yes, fixed 688b7e4b5f7734a1a7a6bc47b735250fcd7614d1